department-of-veterans-affairs/caseflow
# Anya/3285 add more fields to issues

1. Re-name `issues` table to be `worksheet_issues`
2. `Issue` class will not longer correspond to a table
3. Create a `WorksheetIssue` model to represent hearing worksheet issues

Steps To Validate:
1. Pull a hearing from the rails console: h = Hearing.last
2. Run `h.to_hash_for_worksheet`, make sure it returns `worksheet_issues`
3. In the UI, load new hearings for a judge, ensure  `worksheet_issues` table is populated from the appeal issues.
